docs/sections/user_guide/cli/drivers/jedi.rst

```diff
@@ -0,0 +1,101 @@
+``jedi``
+========
+
+The ``uw`` mode for configuring and running the JEDI framework.
```

Comment:
I haven't added any additional documentation resources for users to reference and started with this simple introduction.